File: MessageQueue.Web/app/model/JobsInQueue.ts
module MQPractice.Model {
	
	export class JobsInQueue {
		public createdJobIds: string[];

		public completeJobIds: string[];

		public processingJobIds: string[];

		public errorJobIds: string[];

		public successJobIds: string[];

		public message:string;

		constructor() {
			this.createdJobIds = [];
			this.completeJobIds = [];
			this.processingJobIds = [];
			this.errorJobIds = [];
			this.successJobIds = [];
		}

	}

} 

File: MessageQueue.Api/Services/Details/JobsInQueue.cs
using System.Collections.Generic;
using Newtonsoft.Json;

namespace MessageQueue.Api.Services.Details
{
	public class JobsInQueue
	{
		[JsonProperty("createdJobIds")]
		public List<string> CreatedJobIds { get; set; }
		[JsonProperty("completeJobIds")]
		public List<string> CompleteJobIds { get; set; }
		[JsonProperty("processingJobIds")]
		public List<string> ProcessingJobIds { get; set; }
		[JsonProperty("errorJobIds")]
		public List<string> ErrorJobIds { get; set; }
		[JsonProperty("successJobIds")]
		public List<string> SuccessJobIds { get; set; }

	}
}